<ui version="4.0" >
 <class>icqAddUI</class>
 <widget class="QWidget" name="icqAddUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>119</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QRadioButton" name="icqRadioButton" >
     <property name="text" >
      <string>ICQ number:</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLineEdit" name="icqEdit" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="textLabel1_2" >
     <property name="text" >
      <string>Alternatively, you can search the ICQ Whitepages :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>47</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="searchButton" >
     <property name="text" >
      <string>&amp;Search</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QRadioButton" name="aimRadioButton" >
     <property name="text" >
      <string>AOL screen name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <widget class="QLineEdit" name="aimEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>icqRadioButton</tabstop>
  <tabstop>icqEdit</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>aimRadioButton</tabstop>
  <tabstop>aimEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
